cmd/krel/cmd/release_notes.go

```diff
@@ -66,7 +67,10 @@ permissions to your fork of k/sig-release and k-sigs/release-notes.`,
 }
 
 type releaseNotesOptions struct {
-	tag string
+	tag            string
+	sigReleaseFork string
+	mdFile         string
+	jsonFile       string
```

Comment:
Instead of expecting a filename, how about specifying `outputDir`, with a corresponding `--output-dir` flag?
Then you can default filenames for both the markdown and json output.

Reply:
OK, I've implenented this. Maybe we could add a --format flag like in the old release notes tool :thinking: